achievements/contribubit.achievement.js

```diff
@@ -0,0 +1,33 @@
+var _ = require('lodash');
+var console = process.console;
+
+var TheGodfatherConsigliere = {
+	name: 'TheGodfatherConsigliere',
+	check: function(pullRequest, shall) {
+		if (contributedToKibibit(pullRequest)) {
```

Comment:
replace `contributedToKibibit(pullRequest)` with `_.result(pullRequest, 'organization.username') === 'Kibibit'`

another alternative to the function is `pullRequest.organization && pullRequest.organization === 'kibibit'`, which would have been safe enough.